Circular_kernel_3/include/CGAL/Circular_kernel_3/internal_function_compare_to_right_spherical_kernel.h

```diff
@@ -70,7 +70,7 @@ struct Trait_for_cmp_tgt_theta_0{
   typename SK::FT 
   unsigned_tkz_coeff_normal( const typename SK::Point_3& C,const typename SK::FT& gamma_k) const
   {
-    return - CGAL_NTS sign(gamma_k)*C.y();
+    return - (int)CGAL_NTS sign(gamma_k)*C.y();
```

Comment:
> I have seen that this C-style `(int)` conversion is applied several time, to cast the result of `CGAL::sign`

Yes, because of https://github.com/boostorg/multiprecision/issues/36

Reply:
I agree with the parentheses being wrong.

And I was also wrong about the suggestion of `int(expression)` instead of `(int)expresssion`. I thought `int(expression)` was equivalent to `static_cast` but [*it is wrong*]: it is completely equivalent to the c-style cast `(int)expression`. I will reconsider my practice and use more of `static_cast`.

[*it is wrong*]: https://en.cppreference.com/w/cpp/language/explicit_cast